apps/react/demo/.storybook/preview.ts

```diff
@@ -1,7 +1,7 @@
 import { withThemeByClassName } from "@storybook/addon-themes";
 import type { Preview, ReactRenderer } from "@storybook/react";
 
-import "index.css";
+import "../src/index.css";
```

Comment:
If I don't change this to `../src/`, I get the following storybook build error, @advl do you know why this might be happening, considering this was copied from the boilerplate?

Reply:
I think it may have been something wrong with the lockfile - I rebased and things are working OK with the normal index.css import now.